### PR TITLE
Third 1100+ Words Translation -  Commit for Appendix C-Git-Commands

### PR DESCRIPTION
**Pull request by cherryloua**
This is my third contribution for this project. Please, take some time to review it. Thank you!

### CODE REVIEW

File: book/C-git-commands/1-git-commands.asc [hunk 1]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
 Sa <<_distributed_git#_private_team>> at <<_git_tools#_commit_ranges>> aming tinalakay ang `branchA..branchB` na syntax para gamitin ang `git log` na utos para makita kung ano ang kaibahan sa isang katugong branch sa iba pang branch.
 Sa <<_git_tools#_commit_ranges>> malawak naming tinalakay ito.
 
-In <<_git_tools#_merge_log>> and <<_git_tools#_triple_dot>> we cover using the `branchA...branchB` format and the `--left-right` syntax to see what is in one branch or the other but not in both.
-In <<_git_tools#_merge_log>> we also look at how to use the `--merge` option to help with merge conflict debugging as well as using the `--cc` option to look at merge commit conflicts in your history.
+Sa <<_git_tools#_merge_log>> at <<_git_tools#_triple_dot>> tinalakay natin sa `branchA...branchB` format at ang `--left-right` na syntax para makita kung ano ang nasa isang branch o sa ibang branch pero hindi sa dalawa.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: Please change ``sa `branchA...branchB` format`` to ``ang `branchA...branchB` na format``

File: book/C-git-commands/1-git-commands.asc [hunk 2]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
...
 
-In <<_git_reflog>> we use the `-g` option to view the Git reflog through this tool instead of doing branch traversal.
+Sa <<_git_tools#_merge_log>> tiningnan then natin kung paano gamitin ang `--merge` na opsyon para makatulong sa pag-debug ng merge conflict pati na rin ang `--cc` na opsyon para makita ang pag-merge sa mga conflict na commit sa iyong kasaysayan.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: Please change ``then`` to ``din``

File: book/C-git-commands/1-git-commands.asc [hunk 3]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
...
 
-In <<_git_tools#_signing_commits>> we see how to use `--show-signature` to add a validation string to each commit in the `git log` output based on if it was validly signed or not.
+Sa <<_git_tools#_searching>> ginamit natin ang `-S` at `-L` na mga opsyon para gumawa ng mas pinahusay na mga paghahanap para sa mga anumang pangyayari na pangkasaysayan sa code gaya ng pagtanaw sa kasaysayan ng isang function.
+
+Sa <<_git_tools#_signing_commits>> nakita natin kung paano gamitin ang `--show-signature` para magdagdag ng pagpapatibay na string sa bawat commit sa output ng `git log` nakabase dito kung ito ay wastong pag-sign o hindi.
 
 ==== git stash
 
-The `git stash` command is used to temporarily store uncommitted work in order to clean out your working directory without having to commit unfinished work on a branch.
+Ang `git stash` na utos ay ginamit para pansamantalang pag-imbak sa hindi na commit na trabaho upang luminis ang iyong gumaganang direktoryo na hindi kailangang mag-commit sa di tapos na trabaho sa isang branch.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: ``pansamantalang pag-imbak sa hindi na commit`` to ``pansamantalang mag-imbak ng hindi na-commit``

File: book/C-git-commands/1-git-commands.asc [hunk 4]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
...
-This is basically entirely covered in <<_git_tools#_git_stashing>>.
+It ay pangkalahatang tinalakay sa <<_git_tools#_git_stashing>>.
 
 ==== git tag
 
 The `git tag` command is used to give a permanent bookmark to a specific point in the code history.
 Generally this is used for things like releases.
 
-This command is introduced and covered in detail in <<_git_basics_chapter#_git_tagging>> and we use it in practice in <<_distributed_git#_tagging_releases>>.
+Ang `git tag` na utos ay ginamit para mabigay ng permanenteng bookmark para sa tiyak na punto sa kasaysayan ng code.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: ``mabigay`` to ``magbigay``

File: book/C-git-commands/1-git-commands.asc [hunk 5]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
...
 
-We also cover how to create a GPG signed tag with the `-s` flag and verify one with the `-v` flag in <<_git_tools#_signing>>.
+Ang utos na ito ay pinakilala at tinalakay ng detalye sa <<_git_basics_chapter#_git_tagging>> at ginamit namin ito sa pagsasanay sa <<_distributed_git#_tagging_releases>>.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: `tinalakay ng detalye` to `tinalakay ang detalye`

File: book/C-git-commands/1-git-commands.asc [hunk 6]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
...
+Una naming tiningnan ang utos na ito sa <<_git_basics_chapter#_fetching_and_pulling>> at patuloy nating makikita ang mga halimbawa sa paggamit nito sa <<_git_branching#_remote_branches>>.
 
-We also use it in several of the examples in <<_distributed_git#_contributing_project>>.
+Ginamit din namin ito sa iilang mga halimbawa sa <<_distributed_git#_contributing_project>>.
 
-We use it to fetch a single specific reference that is outside of the default space in <<_github#_pr_refs>> and we see how to fetch from a bundle in <<_git_tools#_bundling>>.
+Ginamit namin ito para kumuha ng nag-iisang tiyak na sanggunian na nasa labas ng default na lugar sa <<_github#_pr_refs>> at nakita din natin kung paano kunin ang nanggaling sa bigkis sa <<_git_tools#_bundling>>.
 
-We set up highly custom refspecs in order to make `git fetch` do something a little different than the default in <<_git_internals#_refspec>>.
+Nagset-up kami ng napakainam na custom refspecs para lang makagawa ang `git fetch` ng bagay na medyo naiiba kaysa sa sanggunian sa <<_git_internals#_refspec>>.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: ``para lang makagawa ang `git fetch` ``  to `` para lang makagawa ng `git fetch` ``

File: book/C-git-commands/1-git-commands.asc [hunk 7]
```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
...
-In <<_git_branching#_delete_branches>> we use the `--delete` flag to delete a branch on the server with `git push`.
+Una nating nakita ang `git push` na utos sa <<_git_basics_chapter#_pushing_remotes>>.
+Dito natalakay natin ang mga pangunahin ng pag-push sa branch para sa remote na repositoryo.
+Sa <<_git_branching#_pushing_branches>> tinalakay natin na mas malalim para sa pag-push ng tiyak na mga branch at sa <<_git_branching#_tracking_branches>> makikita natin paano magset-up ng pagsubaybay ng mga branch para awtomatikong i-push ito.
 
-Throughout <<_distributed_git#_contributing_project>> we see several examples of using `git push` to share work on branches through multiple remotes.
+Sa <<_git_branching#_delete_branches>> ginamit natin ang `--delete` na flag para ibura ang branch sa server sa pamamagitan ng `git push`.
 
-We see how to use it to share tags that you have made with the `--tags` option in <<_git_basics_chapter#_sharing_tags>>.
+Sa buong <<_distributed_git#_contributing_project>> makikita natin ang iilang mga halimbawa sa paggamit ng `git push` para ibahagi ang ginagwa sa mga branch sa pamamaginta ng maraming mga remote.
```
Comment (josteinfrancois) [inline book/C-git-commands/1-git-commands.asc]: `ginagwa` to `ginawa` and `pamamaginta` to `pamamagitan`

**Review by josteinfrancois — COMMENTED** [review]
Hi @cherryloua. I have reviewed your work. Please see the comments I have placed. Thanks.

**Comment by cherryloua** [comment]
Thank you for your PR review @josteinfrancois...  

@severinolorillajr changes are now applied in my latest commit. Thank you!

**Pull request merged**